pkg/generators/models/testdata/cases/validation/api.yaml

```diff
@@ -0,0 +1,62 @@
+openapi: 3.0.0
+info:
+  version: 0.1.0
+  title: Test
+
+components:
+  schemas:
+    Person:
+      type: object
+      required:
+        - name
+        - gender
```

Comment:
I would like to see a nullable required field added to the list

Reply:
added (name in Address)